webextensions/api/declarativeNetRequest.json

```
@@ -969,6 +969,26 @@
               }
             }
           },
+          "isUrlFilterCaseSensitive": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": "84",
+                  "notes": "From 118 default value is false."
```

```suggestion
                  "notes": "The default value changed from true to false in Chrome 118."
```

Phrasing could be better, but I am inverting the message: false is supposedly the standard behavior, so unless stated otherwise we would assume false to be the default.